README.md

```diff
@@ -4,326 +4,117 @@
+## Installing ParlAI
 
-- downloads tasks/datasets on demand and provides the same simple interface to them
-- unifies dataset input and evaluation frameworks/metrics
-- `agents/` directory encourages researchers to submit their training code to the repository to share with others
-- aids reproducibility
+ParlAI currently requires Python3. Dependencies of the core modules are listed in requirement.txt. Some models included (in parlai/agents) have additional requirements.
 
-End goal is general dialogue, which includes many different skills
+Run the following commands to clone the repository and install ParlAI:
 
-- seamlessly combines simulated and real language tasks
-- encourages multi-task model development & evaluation
-- helps to reduce overfitting of models to specific datasets         
+```bash
+git clone https://github.com/facebookresearch/ParlAI.git ~/ParlAI
+cd ~/ParlAI; python setup.py develop
+```
 
-End goal is real dialogue with people
+This will link the cloned directory to your site-packages.
 
-- train and evaluate on live dialogue with humans via Mechanical Turk or Messenger
-- easy setup for connecting turkers with your dialogue agent
-- allow to compare different research groups turk experiments
+This is the recommended installation procedure, as it provides ready access to the examples and allows you to modify anything you might need. This is especially useful if you if you want to submit another task to the repository.
 
-Set of datasets to bootstrap a working dialogue model for human interaction
+All needed data will be downloaded to ~/ParlAI/data, and any non-data files (such as the MemNN code) if requested will be downloaded to ~/ParlAI/downloads. If you need to clear out the space used by these files, you can safely delete these directories and any files needed will be downloaded again.
 
-- motivates building new datasets that will go in the repository
+## Documentation
 
-## Properties
+ - [Basics: world, agents, teachers, action and observations](https://parl.ai/docs/tutorial_basic.html)
+ - [List of available tasks/datasets](https://parl.ai/docs/tasks.html)
+ - [Creating a dataset/task](http://www.parl.ai/docs/tutorial_task.html).
+ - [List of available agents](./parlai/agents)
+ - [Creating a new agent](https://parl.ai/docs/tutorial_seq2seq.html#).
+ - [Model zoo (pretrained models)](https://parl.ai/docs/zoo.html)
+ - [Plug into MTurk](http://parl.ai/docs/tutorial_mturk.html)
+ - [Plug into Facebook Messenger](http://parl.ai/docs/tutorial_messenger.html)
 
-- All datasets look like natural dialogue: a single format / API.
-- Both fixed datasets (conversation logs) and interactive (online/RL) tasks.
-- Both real and simulated tasks.
-- Supports other media, e.g. visual in VQA.
-- Can use Mechanical Turk to run / collect data / evaluate.
-- Python framework.
-- Examples of training with PyTorch.
-- Supports batch and hogwild training and evaluation of models.
 
-## Basic Examples
+## Examples
```

Comment:
Yeah, we talked about having examples that are more similar to what users will likely actually use ParlAI for. Maybe we should just grab the display_data, train_model, and eval_model from the transformer on twitter section.

Reply:
This is a good idea, this is an actual good example